br.eng.silas.qpdftools.yml

```diff
@@ -0,0 +1,45 @@
+id: br.eng.silas.qpdftools
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: qpdftools
+cleanup:
+  - '/include'
+  - '/share/doc'
+  - '/share/man'
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
```

Comment:
why? this the very very very broad.

Reply:
See my other comment. I noticed that the portals had inconsistent behavior. Giving access to the filesystem fix the bug.

Maybe it would work just giving it permission to home folder, but I did not test it. Wait a minute

Reply:
You were right. I limited access to home and it continued to work fine.